templateChecks.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
If it needs bash then in the circleci call above it should be called with it, not with sh.